guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -0,0 +1,365 @@
+
+[[Host_Collection]]
+Host Collection:: A user defined group of one or more Hosts used for bulk actions such as errata installation.
+
+[[Host_Group]]
+Host Group:: A template for building a host.
+Host groups hold shared parameters, such as subnet or lifecycle environment, that are inherited by host group members.
+Host groups can be nested to create a hierarchical structure.
+
+[[Host_Image]]
+Host Image:: A boot disk used for PXE-less provisioning of a specific host.
+The host image only contains the boot files necessary to access the installation media on {ProjectServer}.
+
+[[Incremental_Update]]
+Incremental Upgrade (of a Content View):: The act of creating a new (minor) content view version in a lifecycle environment.
+Incremental upgrades provide a way to make in-place modification of an already published content view.
+Useful for rapid updates, for example when applying security errata.
+
+[[Job]]
+Job:: A command executed remotely on a host from {ProjectServer}.
+Every job is defined in a job template.
+
+[[Job_Template]]
+Job Template:: Defines properties of a job.
+
+[[Katello]]
+Katello:: A Foreman plugin responsible for subscription and repository management.
+
+[[Lazy_Sync]]
+Lazy Sync:: The ability to change the default download policy of a repository from *Immediate* to *On Demand*.
+The *On Demand* setting saves storage space and synchronization time by only downloading the packages when requested by a host.
+
+[[Location]]
+Location:: A collection of default settings that represent a physical place.
+
+[[Library]]
+Library:: A container for content from all synchronized repositories on {ProjectServer}.
+Libraries exist by default for each organization as the root of every lifecycle environment path and the source of content for every content view.
+
+[[Lifecycle_Environment]]
+Lifecycle Environment:: A container for content view versions consumed by the content hosts.
+A Lifecycle Environment represents a step in the lifecycle environment path.
+Content moves through lifecycle environments by publishing and promoting content views.
+
+[[Lifecycle_Environment_Path]]
+Lifecycle Environment Path:: A sequence of lifecycle environments through which the content views are promoted.
+You can promote a content view through a typical promotion path; for example, from development to test to production.
+
+[[Manifest]]
+Manifest (Red{nbsp}Hat Subscription Manifest):: A mechanism for transferring subscriptions from the Red{nbsp}Hat Customer Portal to {ProjectName}.
+Do not confuse with xref:Puppet_Manifest[Puppet Manifest].
+
+[[Migrating_{project-context}]]
+Migrating {Project}:: The process of moving an existing {Project} installation to a new instance.
+
+[[OpenSCAP]]
+OpenSCAP:: A project implementing security compliance auditing according to the Security Content Automation Protocol (SCAP).
+OpenSCAP is integrated in {Project} to provide compliance auditing for hosts.
+
```

Comment:
```suggestion
```
Foreman documentation is not the right place to explain what OpenSCAP is.

Reply:
Disagree because Foreman _uses_ the term. IMHO it's very important to at least have those two sentences. For a complete description, maybe we can refer users to official docs?